<launch>
    <rosparam file="$(find mocap_toolbox)/config/transformation_mocap.yaml" command="load"/>
    <node name="transformation_mocap" pkg="mocap_toolbox" type="transformation_mocap.py" output="screen"/>
</launch>